client/src/components/Pets/EditPetForm/EditPetForm.tsx

```diff
@@ -0,0 +1,327 @@
+                      }}
+                    >
+                      <FormControlLabel value={'true'} control={<Radio className={classes.radioSize} />} label="Yes" />
+                      <FormControlLabel value={'false'} control={<Radio className={classes.radioSize} />} label="No" />
+                    </RadioGroup>
+                  </Grid>
+                </Grid>
+                <Grid container className={classes.inputBox}>
+                  <Grid item xs={12} sm={12} md={4} className={classes.petInputFiledLabelRow}>
+                    <Typography className={classes.label}>Feeding Schedule</Typography>
+                  </Grid>
+                  <Grid item xs={12} sm={12} md={8}>
+                    <TextField
+                      id="feedingSchedule"
+                      variant="outlined"
+                      margin="normal"
+                      className={classes.petInputFiledRow}
+                      name="feedingSchedule"
+                      autoFocus
+                      value={values.feedingSchedule}
+                      onChange={handleChange}
+                      placeholder="eg. Every Morning, twich a day"
+                    />
+                  </Grid>
+                </Grid>
+                <Grid container className={classes.formSpaceB}>
+                  <Grid item xs={12} sm={12} md={4} className={classes.petInputFiledLabelRow}>
+                    <Typography className={classes.label}>About your pet</Typography>
+                  </Grid>
+                  <Grid item xs={12} sm={12} md={8}>
+                    <TextField
+                      id="description"
+                      multiline
+                      rows={6}
+                      rowsMax={6}
+                      className={classes.petInputFiledRow}
+                      variant="outlined"
+                      margin="normal"
+                      name="description"
+                      autoComplete="description"
+                      autoFocus
+                      value={values.description}
+                      onChange={handleChange}
+                      placeholder="tell us about your pet"
+                    />
+                  </Grid>
+                </Grid>
+                <Divider />
+                <Grid container justify="center" className={classes.formSpace}>
+                  <Typography variant={'h3'}>Your Pet Photo Gallery</Typography>
+                </Grid>
+                <Grid container className={classes.petPhotoGalleryRow}>
+                  {petGalleries
+                    ? petGalleries.map((p) => {
+                        return (
+                          <Box key={p.name} className={classes.addNewPetPhotoGallery}>
+                            <img src={URL.createObjectURL(p)} className={classes.petImgSize} />
+                          </Box>
+                        );
+                      })
+                    : ''}
```

Comment:
Though this is okay. But instead of rendering an empty string when `petGalleries` is falsy, I'd prefer rendering only when `petGalleries` is truth like this:
`{!!petGalleries && petGalleries.map(...`